java/streaming/src/main/java/com/ray/streaming/core/processor/MasterProcessor.java

```diff
@@ -0,0 +1,111 @@
+package com.ray.streaming.core.processor;
+
+import com.ray.streaming.api.collector.Collector;
+import com.ray.streaming.core.command.BatchInfo;
+import com.ray.streaming.core.graph.ExecutionGraph;
+import com.ray.streaming.core.graph.ExecutionNode;
+import com.ray.streaming.core.graph.ExecutionNode.NodeType;
+import com.ray.streaming.core.graph.ExecutionTask;
+import com.ray.streaming.core.runtime.context.RuntimeContext;
+import com.ray.streaming.message.Record;
+import com.ray.streaming.operator.impl.MasterOperator;
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.stream.Collectors;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+public class MasterProcessor extends StreamProcessor<BatchInfo, MasterOperator> {
```

Comment:
Add some documentation.

Reply:
@jliagouris I add some documentation, MasterProcessor is same to flink jobManager, it responsible for overall control logic for total dataflow

Reply:
I see. Why not using the same name then? In any case I think we need a list of class names for both APIs to keep them aligned. We'll put together a document for the Python API and it'd be great if you could do the same for your Java API.